core/src/main/java/org/apache/brooklyn/util/core/task/system/ProcessTaskWrapper.java

```diff
@@ -49,10 +47,9 @@
     private final Task<RET> task;
 
     // execution details
-    private ByteArrayOutputStream stdout;
-    private ByteArrayOutputStream stderr;
```

Comment:
i deliberately left `stdout` and `stderr` in case they were needed for deserialization.  but we don't normally persist tasks so this should be fine -- but one to remember.